_posts/en/newsletters/2023-02-08-newsletter.md

```diff
@@ -0,0 +1,130 @@
+    Lightning-Dev mailing list for announcements of upcoming meetings.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+FIXME is a PR by FIXME that FIXME:LarryRuane
+
+{% include functions/details-list.md
+  q0="FIXME"
+  a0="FIXME"
+  a0link="https://bitcoincore.reviews/FIXME#l-FIXME"
+%}
+
+<!-- FIXME:harding to add releases/rc if any on Tuesday -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25880][] p2p: Make stalling timeout adaptive during IBD FIXME:Xekyo
+
+- [Core Lightning #5679][] sql plugin FIXME:adamjonas
+
+- [Core Lightning #5821][] adds `preapproveinvoice` (pre-approve
+  invoice) and `preapprovekeysend` (pre-approve keysend) RPCs that allow
+  the caller to send either a [BOLT11][] invoice or a [keysend][topic
+  spontaneous payments] payment details to Core Lightning's signing module
+  (`hsmd`) to verify the module is willing to sign the payment.  For
+  some applications, such as those where the amount of money which can
+  be spent is rate limited, asking for pre-approval might produce fewer
+  problems than simply attempting the payment and dealing with failure.
+
+- [Core Lightning #5849][] makes backend changes that allow a node to
+  handle over 100,000 peers each with one channel.  Although someone
+  running such a node in production is not likely in the near
+  future---it would take over a dozen entire blocks to just open that
+  many channels---testing the behavior helped the developer make several
+  performance improvements.
+
+- [Core Lightning #5892][] updates CLN's implementation of the
+  [offers][topic offers] protocol based on compatibility testing
+  performed by a developer working on Eclair's implementation.
+
+- [Eclair #2565][] now requests that funds from a closed channel be sent
+  to a new onchain address, rather than an address which was generated
+  when the channel was funded.  This may decrease [output linking][topic
+  output linking], which helps improve user privacy.  An exception to
+  this policy is when the user enables the LN protocol option
+  `upfront-shutdown-script`, which is a request sent to the channel
+  partner at funding time to only use the closing address specified at
+  that time (see [Newsletter #158][news158 upfront] for details).
+
+- [LND #7252][] adds support for using Sqlite as LND's database backend.
```

Comment:
```suggestion
- [LND #7252][] adds support for using SQLite as LND's database backend.
```

Reply:
I opened a for-discussion PR about updating the style guide to have us capitalize proper nouns using standard English rules, see https://github.com/bitcoinops/bitcoinops.github.io/pull/1005 ; for now, I suggest we keep this text as-is.  If we decide against the proposed policy, I'll come back and update this string.